extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/OracleDialectConfig.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.hibernate.orm.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocDefault;
+import io.quarkus.runtime.annotations.ConfigGroup;
+
+/**
+ * Configuration specific to the Hibernate ORM {@linkplain org.hibernate.dialect.OracleDialect}
+ *
+ * @author Steve Ebersole
```

Comment:
Fair warning: `@author` tags are "disallowed" in Quarkus, see here for a rationale: https://github.com/quarkusio/quarkus/blob/34d15fc919adccfda3f9280c4100bd0c052e3881/CONTRIBUTING.md#L190-L193

I'm not sure if they automatically get checked or not, but you'll probably want to get rid of them.

Reply:
> Fair warning: @author tags are "disallowed" in Quarkus, see here for a rationale:

No problem.  My IDE just adds them